src/presets/index.ts

```diff
@@ -14,7 +14,7 @@ export { digitalOcean } from "./digital-ocean";
 export { firebase } from "./firebase";
 export { heroku } from "./heroku";
 export { edgio } from "./edgio";
-export { netlify, netlifyBuilder, netlifyEdge, netlifyStatic } from "./netlify";
+export { netlify, netlifyEdge, netlifyStatic } from "./netlify";
```

Comment:
We need to keep `netlifyBuilder` for backward compatibility for users who might be using it currently